craft_archives/repo/apt_preferences_manager.py

```diff
@@ -0,0 +1,140 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2015-2023 Canonical Ltd.
```

Comment:
I think we can just use `2023` here, no? Does this come from somewhere?

Reply:
The copyright block comes from elsewhere and I forgot to modify the year. Thanks for catching it!